lib/tests/streamlit/delta_generator_test.py

```diff
@@ -243,7 +249,10 @@ def test_constructor_with_id(self):
     def test_enqueue_null(self):
         # Test "Null" Delta generators
         dg = DeltaGenerator(container=None)
-        enqueue_fn = lambda x: None
```

Comment:
Wait, am I crazy or is `enqueue_fn` unused before and after anyways?

Reply:
Seems like it. Might be why my autoformatter removed it.